bbb-record-api/database/create.sql

```diff
@@ -0,0 +1,88 @@
+DROP SCHEMA IF EXISTS `bbb`;

```

Comment:
Better avoid this raw schema and use library to manage the migrations like https://flywaydb.org